mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/InMemoryMailboxManager.java

```diff
@@ -65,6 +66,12 @@ public InMemoryMailboxManager(MailboxSessionMapperFactory mailboxSessionMapperFa
     }
 
     @Override
+    @Inject
+    public void setMessageSearchIndex(MessageSearchIndex index) {
```

Comment:
do you really need this? It is not used in this commit

Reply:
Yes, we have the same part of code in `CassandraMailboxManager`, so I just take it in order to be consistent.
This is a bad way to inject members, but modifying the constructor leads to a lot of changes...